temporalcli/commands.go

```diff
@@ -254,11 +257,21 @@ func (c *CommandContext) populateFlagsFromEnv(flags *pflag.FlagSet) error {
 						flag.Name, anns[0], envVal, err)
 					return
 				}
+				if flag.Changed {
+					logCalls = append(logCalls, func(l *slog.Logger) {
+						l.Info("Env var overrode --env setting", "env_var", anns[0], "flag", flag.Name, "value", envVal)
```

Comment:
```suggestion
						l.Info("Env var overrode --env setting", "env_var", anns[0], "flag", flag.Name)
```

I think we should leave off the value, this can be confidential information and not sure it's needed.

Reply:
Good point